skstore/ts/src/prv/skstore_impl.ts

```diff
@@ -1765,3 +1799,33 @@ function toUpdateWhereQuery(
   const query = `UPDATE "${name}" SET ${sets.query} WHERE ${qWhere.query};`;
   return { query, params: { ...sets.params, ...qWhere.params } };
 }
+
+function check<T>(value: T): void {
+  const type = typeof value;
+  if (type == "string" || type == "number" || type == "boolean") {
+    return;
+  } else if (type == "object") {
+    const jso = value as any;
+    if ((value as any).__sk_frozen) {
+      return;
+    } else if (jso.constructor.name) {
+      throw new Error(
+        "class '" + jso.constructor.name + "' cannot be manage by skstore.",
+      );
+    } else if (Object.isFrozen(jso)) {
```

Comment:
Why is this allowed? Why not only support objects with the tag __sk_frozen?
Is it for convenience when passing arrays?

Reply:
It was just temporary.
I needed to do some testing.
No more the case.